client/src/unauthenticated-app.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react"
+
+export const UnauthenticatedApp = () => {
+  return (
+    <div>
+      <h1>Unauthenticated App</h1>
```

Comment:
Should return 404 or login

Reply:
Yep, will implement in future PRs, since currently have no design of login.

Reply:
Then don't commit this file